src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -10,11 +10,13 @@
 import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 public class ConductorConfig {
     private static final String DefaultConfigFile = "/config.yaml";
+    private static final String CUSTOM_CAPABILITIES = "customCapabilities";
```

Comment:
We have to decide if we want SHOUTY_CAPS or CamelCaps for constants....

Reply:
[SHOUTY_IS_USUALLY_THE_CONVENTION_FOR_CONSTANTS](https://www.safaribooksonline.com/library/view/java-8-pocket/9781491901083/ch01.html#generic_type_parameter_names)

Reply:
Yeah but it's awful.

Reply:
What? You need to speak louder, I_CAN'T_HEAR_WHAT_YOU_ARE_SAYING!